traveler-swift-core/TravelerKit/Traveler.swift

```diff
@@ -680,8 +675,8 @@ public class Traveler {
         A `CancellationError.expiredQuote` will be thrown if the quote has expired.
      */
 
-    public static func cancelOrder(quote: CancellationQuote, delegate: CancellationDelegate) {
-        shared?.cancelOrder(quote: quote, competion: { [weak delegate] (order, error) in
+    public static func cancelOrder(request: CancellationRequest, delegate: CancellationDelegate) {
```

Comment:
My suggestion on this is the following signature:

`func cancelOrder(_ request: CancellationRequest, delegate: CancellationDelegate)`

Lets take votes? @OmarPadierna @benRmc

Reply:
I agree.

Reply:
How about cancelOrderWith(_ request, ...)?

Reply:
I still prefer the above